<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Учёт ремонтной мастерской</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(81, 0, 135, 255), stop:0.427447 rgba(41, 61, 132, 235), stop:1 rgba(155, 79, 165, 255));
font-family: Noto Sans SC;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QFrame" name="balance_frame">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 30);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 7px;</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>12</number>
         </property>
         <property name="topMargin">
          <number>12</number>
         </property>
         <property name="rightMargin">
          <number>12</number>
         </property>
         <property name="bottomMargin">
          <number>12</number>
         </property>
         <item>
          <widget class="QLabel" name="current_balance">
           <property name="styleSheet">
            <string notr="true">color: white;
font-weight: bold;
font-size: 20pt;
background-color: none;
border: none;</string>
           </property>
           <property name="text">
            <string>Общая сумма</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="balance">
           <property name="styleSheet">
            <string notr="true">color: white;
font-size: 30pt;
background-color: none;
border: none;</string>
           </property>
           <property name="text">
            <string>0 руб.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="all">
           <property name="styleSheet">
            <string notr="true">color: white;
font-size: 30pt;
background-color: none;
border: none;</string>
           </property>
           <property name="text">
            <string>всего заказов: 0</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: none;
border: none;
padding-top: 10px;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="res.qrc">:/icon/icons/solved.svg</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="income">
             <property name="styleSheet">
              <string notr="true">color: white;
font-size: 16pt;
background-color: none;
border: none;
font-weight: bold;
padding-top: 10px;</string>
             </property>
             <property name="text">
              <string>Выполнено</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="solved_balance">
           <property name="styleSheet">
            <string notr="true">color: white;
font-size: 20pt;
background-color: none;
border: none;</string>
           </property>
           <property name="text">
            <string>0 руб.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="count_solved">
           <property name="styleSheet">
            <string notr="true">color: white;
font-size: 20pt;
background-color: none;
border: none;</string>
           </property>
           <property name="text">
            <string>кол-во заказов: 0</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: none;
border: none;
padding-top: 10px;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="res.qrc">:/icon/icons/not_solved.svg</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="outcome">
             <property name="styleSheet">
              <string notr="true">color: white;
font-size: 16pt;
background-color: none;
border: none;
font-weight: bold;
padding-top: 10px;</string>
             </property>
             <property name="text">
              <string>Не выполнено</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="not_solved_balance">
           <property name="styleSheet">
            <string notr="true">color: white;
font-size: 20pt;
background-color: none;
border: none;</string>
           </property>
           <property name="text">
            <string>0 руб.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="count_not_solved">
           <property name="styleSheet">
            <string notr="true">color: white;
font-size: 20pt;
background-color: none;
border: none;</string>
           </property>
           <property name="text">
            <string>кол-во заказов: 0</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="categories_frame">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 30);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 7px;</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>12</number>
         </property>
         <property name="topMargin">
          <number>12</number>
         </property>
         <property name="rightMargin">
          <number>12</number>
         </property>
         <property name="bottomMargin">
          <number>12</number>
         </property>
         <item>
          <widget class="QLabel" name="categories">
           <property name="styleSheet">
            <string notr="true">color: white;
font-weight: bold;
font-size: 20pt;
background-color: none;
border: none;</string>
           </property>
           <property name="text">
            <string>Виды ремонта</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: none;
border: none;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="res.qrc">:/icon/icons/phone.svg</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="phone">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: white;
font-weight: bold;
font-size: 14pt;
background-color: none;
border: none;
width: 80;</string>
             </property>
             <property name="text">
              <string>Ремонт
телефона</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="balance_phone">
             <property name="styleSheet">
              <string notr="true">color: white;
font-size: 16pt;
background-color: none;
border: none;</string>
             </property>
             <property name="text">
              <string>0 руб.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="count_phone">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: white;
font-size: 16pt;
background-color: none;
border: none;</string>
             </property>
             <property name="text">
              <string>кол-во: 0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_15">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: none;
border: none;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="res.qrc">:/icon/icons/tablet.svg</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="tablet">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: white;
font-weight: bold;
font-size: 14pt;
background-color: none;
border: none;
width: 80;</string>
             </property>
             <property name="text">
              <string>Ремонт
планшета</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="balance_tablet">
             <property name="styleSheet">
              <string notr="true">color: white;
font-size: 16pt;
background-color: none;
border: none;</string>
             </property>
             <property name="text">
              <string>0 руб.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="count_tablet">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: white;
font-size: 16pt;
background-color: none;
border: none;</string>
             </property>
             <property name="text">
              <string>кол-во: 0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_18">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="sizeIncrement">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: none;
border: none;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="res.qrc">:/icon/icons/labtop.svg</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labtop">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: white;
font-weight: bold;
font-size: 14pt;
background-color: none;
border: none;
width: 80;</string>
             </property>
             <property name="text">
              <string>Ремонт
ноутбука</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="balance_labtop">
             <property name="styleSheet">
              <string notr="true">color: white;
font-size: 16pt;
background-color: none;
border: none;</string>
             </property>
             <property name="text">
              <string>0 руб.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="count_labtop">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: white;
font-size: 16pt;
background-color: none;
border: none;</string>
             </property>
             <property name="text">
              <string>кол-во: 0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_21">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: none;
border: none;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="res.qrc">:/icon/icons/computer.svg</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="computer">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: white;
font-weight: bold;
font-size: 14pt;
background-color: none;
border: none;
width: 80;</string>
             </property>
             <property name="text">
              <string>Ремонт
компьютера</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="balance_computer">
             <property name="styleSheet">
              <string notr="true">color: white;
font-size: 16pt;
background-color: none;
border: none;</string>
             </property>
             <property name="text">
              <string>0 руб.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="count_computer">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: white;
font-size: 16pt;
background-color: none;
border: none;</string>
             </property>
             <property name="text">
              <string>кол-во: 0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_22">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: none;
border: none;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="res.qrc">:/icon/icons/other.svg</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="other">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: white;
font-weight: bold;
font-size: 14pt;
background-color: none;
border: none;
width: 80;</string>
             </property>
             <property name="text">
              <string>Ремонт
другой
техники</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="balance_other">
             <property name="styleSheet">
              <string notr="true">color: white;
font-size: 16pt;
background-color: none;
border: none;</string>
             </property>
             <property name="text">
              <string>0 руб.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="count_other">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: white;
font-size: 16pt;
background-color: none;
border: none;</string>
             </property>
             <property name="text">
              <string>кол-во: 0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QPushButton" name="new_transaction">
        <property name="styleSheet">
         <string notr="true">QPushButton {
color: white;
background-color: rgba(255, 255, 255, 30);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 7px;
width: 230px;
height: 50px;
}
QPushButton:hover {
background-color: rgba(255, 255, 255, 40);
}
QPushButton:pressed {
background-color: rgba(255, 255, 255, 70);
}</string>
        </property>
        <property name="text">
         <string>Новый заказ</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icon/icons/city.svg</normaloff>:/icon/icons/city.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>240</width>
          <height>240</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="edit_transaction">
        <property name="styleSheet">
         <string notr="true">QPushButton {
color: white;
background-color: rgba(255, 255, 255, 30);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 7px;
width: 230px;
height: 50px;
}
QPushButton:hover {
background-color: rgba(255, 255, 255, 40);
}
QPushButton:pressed {
background-color: rgba(255, 255, 255, 70);
}</string>
        </property>
        <property name="text">
         <string>Изменить заказ</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icon/icons/city.svg</normaloff>:/icon/icons/city.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delete_transaction">
        <property name="styleSheet">
         <string notr="true">QPushButton {
color: white;
background-color: rgba(255, 255, 255, 30);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 7px;
width: 230px;
height: 50px;
}
QPushButton:hover {
background-color: rgba(255, 255, 255, 40);
}
QPushButton:pressed {
background-color: rgba(255, 255, 255, 70);
}</string>
        </property>
        <property name="text">
         <string>Отменить заказ</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icon/icons/city.svg</normaloff>:/icon/icons/city.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="styleSheet">
       <string notr="true">QTableView {
background-color: rgba(255, 255, 255, 30);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 7px;
}
QTableView::section {
background-color: rgba(53, 53, 53);
color: white;
border: none;
height: 50px;
font-size: 14pt;
}
QTableView::item {
border-style: none;
border-bottom: rgba(255, 255, 255, 50);
}
QTableView::item:selected {
border: none;
//color: rgba(255, 255, 255);
//background-color: rgba(255, 255, 255, 50);
}</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>135</number>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
